src/highlighters.cc

```diff
@@ -1116,10 +1125,10 @@ struct LineNumbersHighlighter : Highlighter
         const Face face = faces["LineNumbers"];
         const Face face_wrapped = faces["LineNumbersWrapped"];
         const Face face_absolute = faces["LineNumberCursor"];
-        int digit_count = compute_digit_count(context.context);
+        int number_width = std::max(compute_digit_count(context.context), m_min_width);
```

Comment:
its compute_digit_counts that should do the std::max operation. And rename min_width to min_digits.

Reply:
Editing 'compute_digit_counts` was my first guess, but it was `static` and therefore didn't have access to `m_min_digits`, and I didn't know how important static-ness was.

I've changed it to be not-static, and moved all the `std::max()` calls into it.